projects/js-packages/connection/components/disconnect-dialog/steps/step-disconnect.jsx

```diff
@@ -30,8 +30,18 @@ const StepDisconnect = props => {
 		disconnectingPlugin,
 		closeModal,
 		context,
+		trackModalClick,
 	} = props;
 
+	const trackLearnClick = useCallback( () => trackModalClick( 'learn_about' ), [
+		trackModalClick,
+	] );
+	const trackSupportClick = useCallback( () => trackModalClick( 'support' ), [ trackModalClick ] );
+	const trackStayConnectedClick = useCallback( () => {
```

Comment:
Bit of a nitpick - thoughts on naming this callback to something like `handleStayConnectedClick` since it both captures an event and closes the modal?